src/platforms/stm32/src/main.c

```diff
@@ -231,6 +232,7 @@ int main()
     GlobalContext *glb = globalcontext_new();
 
     usart_setup(glb);
+    AVM_LOGD(TAG, "Free Heap before init: %i", MemBegin);
```

Comment:
Do we want to keep these debug statements?

Reply:
The default log level is 'normal' so the messages won't be included for default builds. I originally thought it might be helpful for debugging memory usage if the build is configured for debug logging, especially for boards with less RAM, but after comparing with memory usage reported by the application early in startup and just before exit there isn't much value in keeping them.

Reply:
They are gone now.